.github/workflows/ui-e2e.yaml

```diff
@@ -29,10 +29,11 @@ jobs:
     secrets:
       credentials: ${{ secrets.GCP_CREDENTIALS }}
     with:
-      ui_account: user
+      #ui_account: user
```

Comment:
Not very useful to keep something like this in a comment.

Reply:
Useful because needed to make the test pass with elemental-ui 1.0.0.
I have to check tomorrow why RKE does not work anymore with latest UI and then, the ui_account variable will be added back. I don't want to mix upgrade issue with latest elemental UI adaptations.

Reply:
Here is the spotted issue (thanks automation :smile: ) https://github.com/rancher/elemental-ui/issues/64